app/Http/Controllers/KidAppController.php

```diff
@@ -10,9 +10,33 @@
 use App\Models\ContentType;
 use Illuminate\Http\Request;
 use Illuminate\Support\Facades\Storage;
+use Validator;
+use Illuminate\Support\Facades\Crypt;
 
 class KidAppController extends Controller
-{
+{ 
+    public function register()
+    { 
+       
+        return view('user/register');
+    }
+
+    public function store_register(Request $request)
+    {
+        $user = User::create([
+            'name'=> $request->input('name'),
+            'email' => $request->input('email'),
+            'password' =>Crypt::encrypt($request->input('password'))
```

Comment:
```suggestion
            'password' => Crypt::encrypt($request->input('password'))
```

Er mist hier een spatie